manifests/y/Yourkit/JavaProfiler/2022.3/Yourkit.JavaProfiler.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Yourkit.JavaProfiler
+PackageVersion: 2022.3

```

```suggestion
PackageVersion: 2022.3.96
```
This must match what is shown in control panel. When you update this field, remember to change the name of the version folder to match